README.md

```diff
@@ -1,2 +1,32 @@
 # buttercms-pelican
 ButterCMS Plugin for Pelican getpelican.com
+
+## Requirements
+- buttercms-python
+
+
+## How to use
+
+You need to clone this repo into your plugin folders, then add "buttercms-pelican" to plugins list in `pelicanconf.py`
+
+```
+PLUGIN_PATHS = ["/path/to/plugins"]
+PLUGINS = [sitemap", "buttercms-pelican"]
```

Comment:
```suggestion
PLUGINS = ["sitemap", "buttercms-pelican"]
```

Reply:
Actually, what is "sitemap"?  I don't think it's part of a default project. If it's not then let's not include it in the README.